packages/daemon/src/app.ts

```diff
@@ -0,0 +1,8 @@
+import DB from './boot/database';
+
+module.exports = app => {
+  app.beforeStart(async () => {
+    const db = (await app.applicationContext.getAsync('pipcookDB')) as DB;
+    db.connect();
```

Comment:
What's the reason to move the db connection here instead of @init decorator?

Reply:
@init decorator depend on the dependencies of IOC object, but now, the model apis are offered by static class function like `Pipeline.findAll`, there are no dependencies between Model and Service, so the `DB.connect` never be called though @init.

Reply:
And, before this PR, the Models are initialized and call `model.sync` every request, it's unnecessary.

Reply:
Ok, reasonable, thanks.